compiler/circle-part-value-test/part_eval_one.py

```diff
@@ -23,6 +24,30 @@
 tflite_model = args.name + ".tflite"
 circle_model = args.name + ".circle"
 partition_conn_ini = args.name + ".conn.ini"
+partition_conn_json = args.name + ".conn.json"
+expected_count = args.name + ".excnt"
+
+# Check expected count of models from partitioning
+try:
+    with open(expected_count, "r") as expected_count_file:
+        expected_count_line = expected_count_file.readline()
+
+    expected_count_line = int(expected_count_line)
+    if expected_count_line:
+        with open(partition_conn_json) as json_file:
+            json_data = json.load(json_file)
+            parts_value = json_data["parts"]
+            if len(parts_value) != expected_count_line:
+                print("Partitioned model count differs from expected:",
+                      expected_count_line)
+                quit(255)
```

Comment:
How about using internal Exception class?
```suggestion
                raise ValueError("Partitioned model count differs from expected:",
                      expected_count_line)
```
Or `AssertionError` could be used.

Reply:
Ah, just want to exit with 255 code. This may work but it is tested so I want to go as-is.